### ISSUE TITLE
How can I style the default clear button and make it a bit larger?

### ISSUE THREAD

**Issue by Fa1z-khan** [sanusart/react-dropdown-select#309]

**Comment by sanusart** [comment]
https://sanusart.github.io/react-dropdown-select/prop/clear-renderer/

**Comment by Fa1z-khan** [comment]
yep i got to that part but cant figure out why it doesnt clear while clicking on the custom icon

**Comment by sanusart** [comment]
Its a building blocks when custom renderers used so you need to call clearAll from provided methods.
Like here
https://codesandbox.io/p/sandbox/react-dropdown-select-forked-xv97wc?file=%2Fsrc%2Findex.js%3A20%2C49
